module.nix

```diff
@@ -17,6 +17,13 @@ in
       '';
       default = [ ];
     };
+    extraPostVM = lib.mkOption {
+      type = lib.types.str;
+      description = ''
+        extra shell code to execute once the disk image(s) have been succesfully created and moved to $out
+      '';
+      default = "";
+    };
```

Comment:
an example for the compression would be nice here? :)

Reply:
Done